src/styles/blocks/style-block-header.scss

```diff
@@ -0,0 +1,20 @@
+.header {
+  background: hsl(0, 0%, 100%);
+  box-shadow: 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo-img {
+    display: block;
+    width: 40px;
+    height: 40px;
+    grid-column: 1 / 2;
+    @include transition (all, 300ms);
```

Comment:
Replace 'all' with specific properties for better performance in the transition mixin (e.g. 'transform').